cypress/plugins/addAuth.js

```diff
@@ -0,0 +1,21 @@
+const fs = require("fs");
+const os = require("os");
+const path = require("path");
+const axios = require("axios");
+
+axios
+  .post("https://wroom.io/authentication/signin", {
```

Comment:
hmm, wroom.io isn't available to the pubic, maybe the url should be an env var, or depend on a "mode" flag?

Reply:
what do you mean by that? The wroom.io is mentioned in quite a lot of places within the code (51 instances) 
<img width="431" alt="image" src="https://user-images.githubusercontent.com/69144671/183656663-d6eccca8-1ca4-40ba-83b6-0d00ca1cdd26.png">

Reply:
yes, it's for prismic developers, our dev env is wroom.test, staged is wroom.io. and production is prismic.io.

Normal users don't have access to wroom.io (although, one managed it once but it should be blocked off to the public)